src/styles/index.scss

```diff
@@ -1,3 +1,62 @@
-body {
+@import './variables';
+@import './body';
+@import './grade';
+@import './price';
+
+p {
   margin: 0;
 }
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__items {
+  object-fit: contain;
+  margin: 32px 19px 40px;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  text-align: left;
+  margin: 0 16px 4px;
+  color: $main-text-color;
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  text-align: left;
+  color: #616070;
+  margin: 0 0 16px;
+  padding-left: 16px;
```

Comment:
Padding is applied only to the left side, which can cause inconsistency in the design if the `text-align` is changed or if the layout needs to be adjusted. Consider if padding should be uniform or if additional padding is needed on other sides for better alignment and consistency.